lib/housing-wanted/housing-wanted.models.js

```diff
@@ -0,0 +1,45 @@
+const Joi = require('joi');
+const propertyModels = require('../property/property.models.js');
+
+module.exports = {
+  db: (sequelize, Sequelize) => {
+    const HousingWanted = sequelize.define('HousingWanted', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      author: Sequelize.STRING,
+      title: Sequelize.STRING,
+      location: Sequelize.STRING,
```

Comment:
please remove. should be an association, not defined this way.

Reply:
If i'm understanding this correctly, author, title, and location are not defined in housing-wanted.models because they're already defined in a different module. So when we make the association the definitions for author, location, and title are also linked to the housing-wanted module?